Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN GOARCH=$TARGETARCH go install -ldflags="\
       -X tailscale.com/version.gitCommitStamp=$VERSION_GIT_HASH" \
       -v ./cmd/tailscale ./cmd/tailscaled ./cmd/containerboot
 
-FROM alpine:3.16
+FROM alpine:3.18
```

Comment:
Not sure if anyone uses this image. Our build infra builds with mkctr via ./build_docker.sh script

Reply:
I don't think so. We've discussed removing it a few times but never actually did.